roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/controller/CarController.java

```diff
@@ -43,4 +52,26 @@ public static Car getCar(ICar car) {
 
         return iCarToCarMap.get(car);
     }
+
+    public static void addLostCar(IConsumer consumer, ICar car) {
+        if (!lostCarsMap.containsKey(consumer)) {
+            lostCarsMap.put(consumer, new LinkedList<>());
+        }
+        lostCarsMap.get(consumer).add(car);
```

Comment:
do we need a map here? or would a counter be enough? because objects in maps will not be deleted by garbage collector